app/controllers/api/policies_controller.rb

```diff
@@ -3,5 +3,57 @@ class PoliciesController < BaseController
     include Subcollections::Conditions
     include Subcollections::Events
     include Subcollections::PolicyActions
+
+    def create_resource(type, _id, data = {})
+      assert_id_not_specified(data, type)
+      create_policy(data)
+    end
+
+    def edit_resource(type, id = nil, data = {})
+      raise BadRequestError, "Must specify an id for editing a #{type} resource" unless id
+      policy = resource_search(id, type, collection_class(:policies))
+      add_associations(data, policy)
```

Comment:
how does this work on edits ?  if we're always adding conditions and miq_policy_contents how can we edit some of the currently existing ones ?